packages/cli-lib/lang/en.lyaml

```diff
@@ -441,9 +441,9 @@ en:
           logs:
             describe: "Get logs for a function within a project"
             errors:
-              appPathRequired: "You must specify an app path using the --appPath argument."
               functionNameRequired: "You must pass a function name to retrieve logs for."
-              logs: "No logs were found for the function name \"{{ functionName }}\" in the app path \"{{ appPath }}\" within the project \"{{ projectName }}\" in account {{ accountId }}."
+              noAppFunctionLogs: "No logs were found for the function name \"{{ functionName }}\" in the app path \"{{ appPath }}\" within the project \"{{ projectName }}\" in account {{ accountId }}."
+              noEndpointLogs: "No logs were found for the function name \"{{ functionName }}\" within the project \"{{ projectName }}\" in account {{ accountId }}."
```

Comment:
@erincouse was hoping to get your feedback on these errors. They're shown when no logs are found

cc @kemmerle